doc/api/n-api.md

```diff
@@ -2931,11 +2931,12 @@ NAPI_EXTERN napi_status napi_cancel_async_work(napi_env env,
 
 Returns `napi_ok` if the API succeeded.
 
-This API cancels a previously allocated work, provided
-it has not yet been queued for execution. After this function is called
+This API cancels a previously queued work if it has not yet
```

Comment:
Nit: `a previously queued work`? Should that just be `previously queued work` without the `a`?

Nit: `previously queued` -> `previously-queued` or (even better IMO) just `queued`